buildenv/jenkins/docker-slaves/jenkinsfile-build-container

```diff
@@ -66,9 +68,9 @@ if("${OS}" == "unknown" || !(validArch.contains(ARCH))) {
                     }
                     stage("Push") {
                         withCredentials([usernamePassword(credentialsId: '7fb9f8f0-14bf-469a-9132-91db4dd80c48', passwordVariable: 'PASS', usernameVariable: 'USER')]) {
-                            sh "docker login --username=\"${USER}\" --password=\"${PASS}\""
+                            sh "docker login ${login_info}"
```

Comment:
I think login_info will always be the same, but the username/password will change so we'll need to change the CredentialsID based on param. Also, we'll need a new variable here for the Docker repo url. Default would be blank/nothing since DockerHub is the default. But we should be able to override it with a parameter when we want to push to a different Docker repo.

Reply:
Hey @AdamBrousseau! Thanks for your advice. Is the Docker repo url same as the REPOSITORY we define after the if statement to check OS and ARCH?

Reply:
No I think it's unrelated

Reply:
The file was changed. Thanks for checking! :) @AdamBrousseau